apps/app-article-server/src/browser/components/article-content.js

```diff
@@ -211,8 +212,8 @@ class Content extends Component {
           <div
             className={`expand ${this.props.darkModeEnabled ? "darkMode" : ""}`}
             id={"expandFactBox-" + key}
-            onClick={(ev) => {
-              this.expandFactBox(ev);
+            onClick={() => {
+              this.expandFactBox(key);
             }}
```

Comment:
I reverted this code to check if the old code worked. It's an open question whether we want to revert it back or not. (Applies to line 273 as well).